src/fights.ts

```diff
@@ -1828,10 +1564,22 @@ function deliverThesis(): void {
     outfit("checkpoint");
   }
   cliExecute("gain 1800 muscle");
+
+  let thesisLocation = $location`Uncle Gator's Country Fun-Time Liquid Waste Sluice`;
```

Comment:
Since the user might not have access to barf now, have to find another place to deliver thesis